pkg/events/event.go.go

```diff
@@ -0,0 +1,114 @@
+package events
+
+import (
+	"github.com/aquasecurity/tracee/types/trace"
+)
+
+type Event struct {
```

Comment:
This seems a bit confusing to me, because we will have `events.Event` and `trace.Event`, but one one is the event, the other is the definition of the event. The previous name seemed to be more appropriate `EventDefinition`. WDYT?